lib/maze/capabilities.rb

```diff
@@ -12,7 +12,7 @@ def for_browser_stack_device(device_type, local_id, appium_version, capabilities
           'browserstack.console' => 'errors',
           'browserstack.localIdentifier' => local_id,
           'browserstack.local' => 'true',
-          'disabledAnimations' => 'true',
+          'disableAnimations' => 'true',
```

Comment:
I've been noticing Appium warnings about strings being passed instead of booleans, e.g.
`Capability 'noReset' changed from string to boolean. This may cause unexpected behavior`

```suggestion
          'disableAnimations' => true,
```